packages/teams-js/src/public/appInstallDialog.ts

```diff
@@ -24,8 +27,19 @@ export namespace appInstallDialog {
       if (!isSupported()) {
         throw new Error('Not supported');
       }
-      sendMessageToParent('appInstallDialog.openAppInstallDialog', [openAPPInstallDialogParams]);
-      resolve();
+      if (runtime.isLegacyTeams) {
+        resolve(
+          sendAndHandleError(
+            'executeDeepLink',
+            `${teamsDeepLinkProtocol}://${teamsDeepLinkHost}${teamsDeepLinkUrlPathForAppInstall}${encodeURIComponent(
```

Comment:
Do you think it's worth putting this into deepLinkUtilities as well?

Reply:
Wasn't sure since it is just one component that is being translated. But I think in terms of future proofing it might be beneficial, so I will make that change.